### PR TITLE
Master l10n de din improv bfr

### PR DESCRIPTION
**Pull request by bfr-o**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_de/report/din5008_report.xml [hunk 1]
```diff
@@ -45,6 +45,27 @@
                 <div class="article din" t-att-data-oe-model="o and o._name" t-att-data-oe-id="o and o.id">
                     <div class="header_address">
                         <t t-if="address">
+                            <div class="company_invoice_address">
+                                <t t-if="company.street">
```
Comment (jco-odoo) [inline addons/l10n_de/report/din5008_report.xml:49]: are you sure this needs to be company.street?

**Review by jco-odoo — COMMENTED** [review]
@bfr-o Not sure if this one was on purpose.  Are the bullet special characters according to the standard?  (or it was done like this somewhere else?)  I think it is better to squash the 2 commits and it can go like this into master?

File: addons/l10n_de/report/din5008_report.xml (hunk 1 above)
Comment (bfr-o) [reply addons/l10n_de/report/din5008_report.xml]: I changed that

**Comment by bfr-o** [comment]
@jco-odoo I made the requested changes and squashed the commit with better commit message.

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**